long-term-storage-aws.md

```diff
@@ -57,3 +57,13 @@ as described here: [https://docs.aws.amazon.com/eks/latest/userguide/specify-ser
 Once that annotation has been created and set, you'll need to attach it to the Prometheus pod, the Thanos compact pod, and the Thanos store pod.
 For prometheus, set .Values.prometheus.serviceAccounts.server.create to false, and .Values.prometheus.serviceAccounts.server.name to the name of your created service account
 For thanos set `.Values.thanos.compact.serviceAccount`, and `.Values.thanos.store.serviceAccount` to the name of your created service account.
+
+
+
+__Thanos Encryption With s3 and KMS__
+
+You can encrypt the s3 bucket where kubecost data is stored in AWS via s3 and KMS. However, because thanos can store potentially millions of objects, it is suggested that you use bucket-level encryption instead of object-level encryption. More details available here:
```

Comment:
```suggestion
You can encrypt the S3 bucket where Kubecost data is stored in AWS via S3 and KMS. However, because Thanos can store potentially millions of objects, it is suggested that you use bucket-level encryption instead of object-level encryption. More details available here:
```